go/vt/tabletserver/queryctl.go

```diff
@@ -54,6 +58,7 @@ func init() {
 	flag.IntVar(&qsConfig.RowCache.Connections, "rowcache-connections", DefaultQsConfig.RowCache.Connections, "rowcache max simultaneous connections")
 	flag.IntVar(&qsConfig.RowCache.Threads, "rowcache-threads", DefaultQsConfig.RowCache.Threads, "rowcache number of threads")
 	flag.BoolVar(&qsConfig.RowCache.LockPaged, "rowcache-lock-paged", DefaultQsConfig.RowCache.LockPaged, "whether rowcache locks down paged memory")
+	flag.StringVar(&qsConfig.DB, "db", DefaultQsConfig.DB, "database type, e.g. mysql")
```

Comment:
Look the code again, I feel that the "db" option is better to be placed in go/cmd/vttablet/vttablet.go since other services (e.g. binlog) also need a dbconnection.  Please share your thoughts.